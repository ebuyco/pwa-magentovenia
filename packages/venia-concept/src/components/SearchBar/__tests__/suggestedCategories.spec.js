import React from 'react';
import TestRenderer from 'react-test-renderer';

import SuggestedCategories from '../suggestedCategories';

jest.mock('src/classify');
jest.mock('src/drivers', () => ({
    Link: ({ children }) => children,
    resourceUrl: jest.fn()
}));

const defaultProps = {
    handleCategorySearch: jest.fn(),
    autocompleteQuery: 'query',
    categorySuggestions: [
        {
            value_string: 'foo',
            label: 'label'
        }
    ]
};

test('renders a suggestedProduct component', () => {
    const component = TestRenderer.create(
        <SuggestedCategories {...defaultProps} />
    );

    expect(component.toJSON()).toMatchSnapshot();
});
