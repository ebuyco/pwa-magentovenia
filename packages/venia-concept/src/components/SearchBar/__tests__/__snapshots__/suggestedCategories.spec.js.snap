// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a suggestedProduct component 1`] = `
<ul
  className="root"
>
  <li
    className="item"
  >
    <strong>
      query
    </strong>
     in 
    label
  </li>
</ul>
`;
